modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+[id='virt-troubleshooting-vm-import_{context}']
+= Troubleshooting a virtual machine import
+
+[id='logs_{context}']
+== Logs
+
+ifdef::virt-importing-vmware-vm[]
+You can check the V2V Conversion Pod log for errors.
+
+.Procedure
+
+. Obtain the V2V Conversion Pod name:
++
+----
+$ oc get pods -n <project> | grep v2v <1>
+kubevirt-v2v-conversion-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the V2V Conversion Pod log:
++
+----
+$ oc logs kubevirt-v2v-conversion-f66f7d-zqkz7 -f -n <project>
+----
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+You can check the VM Import Controller Pod log for errors.
+
+.Procedure
+
+. Obtain the VM Import Controller Pod name:
++
+----
+$ oc get pods -n <project> | grep import <1>
+vm-import-controller-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the VM Import Controller Pod log:
++
+----
+$ oc logs vm-import-controller-f66f7d-zqkz7 -f -n <project>
+----
+endif::[]
+
+[id='error-messages_{context}']
+== Error messages
+
+The following error messages might appear:
```

Comment:
Nit re: "error messages": there's currently only one error message listed in the VMWare assembly.

Reply:
I thought it was like "Prerequisites", where we use the plural form even if there is only one. I can change for the VMware section.